Deleted empty sections + added a line in contributions (#3)
barriebyron/www: docs/guide/contributing.rst

Where to start
--------------

The guide has its origins in the efforts of developers to understand and explain documentation to each other. Since then, Write the Docs has expanded its focus and its community. We've recently reorganized the guide to provide a clearer top-level structure, but there's much work to be done. Feel free to suggest further changes to the organization. Here's where we need your contributions:

* More :ref:`api-documentation` content!

> * Helping non-writer roles understand the moving parts of good docs

Reviewer: ```suggestion
* Helping non-writers understand what makes good documentation
```

> * Helping writers new to developer tools or docs-as-code understand the moving parts of new toolchains and workflows

Reviewer: ```suggestion
* Helping writers new to developer tools or docs-as-code workflows to understand these toolchains and workflows
```

> * High-level discussions of common tools and toolchains. Please avoid advocacy here! But thoughtful presentation of specific use cases, how you solved them, what worked well, and what didn't work so well, are most welcome. If you want to write this kind of topic, consider starting with an issue in GitHub, or contact one of the new guide editors.

Reviewer: ```suggestion
* High-level discussions of common tools and toolchains (please avoid advocacy). Present problems, how you solved them, what worked, and what didn't. To write this kind of topic, consider starting with an issue in GitHub or contact one of the guide editors.
```
